src/compiler/binder.ts

```diff
@@ -1398,6 +1400,15 @@ namespace ts {
             }
         }
 
+        function bindGlobalModuleExportDeclaration(node: GlobalModuleExportDeclaration) {
+            if (!file.externalModuleIndicator) {

```

Comment:
Sorry, I meant use `isExternalModule` in `utilities.ts`. Vlad mentioned it offhandedly and I was too lazy to look it up.